docs/src/main/asciidoc/security-authorization-of-web-endpoints-reference.adoc

```diff
@@ -8,55 +8,68 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 include::_attributes.adoc[]
 :categories: security,web
 
-Quarkus has an integrated pluggable web security layer. If security is enabled all HTTP requests will have a permission
-check performed to make sure they are allowed to continue.
+Quarkus has an integrated pluggable web security layer. If security is enabled, all HTTP requests will have a permission check performed to make sure they are allowed to continue.
+This means you can not use `@PermitAll` to open up a path if the path has been blocked by using the `quarkus.http.auth.` configuration.
 
-NOTE: Configuration authorization checks are executed before any annotation-based authorization check is done, so both
-checks have to pass for a request to be allowed. This means you cannot use `@PermitAll` to open up a path if the path has
-been blocked using `quarkus.http.auth.` configuration. If you are using JAX-RS you may want to consider using the
-`quarkus.security.jaxrs.deny-unannotated-endpoints` or `quarkus.security.jaxrs.default-roles-allowed` to set default security
-requirements instead of HTTP path level matching, as these properties can be overridden by annotations on an individual
-endpoint.
+[NOTE]
+====
+If you are using JAX-RS, consider using `quarkus.security.jaxrs.deny-unannotated-endpoints` or `quarkus.security.jaxrs.default-roles-allowed` to set default security requirements instead of HTTP path level matching because annotations can override these properties on an individual endpoint.
+====
 
-Authorization is based on user roles that are provided by the security provider.
+Authorization is based on user roles that the security provider provides.
 To customize these roles, a `SecurityIdentityAugmentor` can be created, see
 xref:security-customization.adoc#security-identity-customization[Security Identity Customization].
 
-== Authorization using Configuration
+== Authorization using configuration
+
+Permissions are defined in the Quarkus configuration using permission sets, with each permission set specifying a policy for access control.
 
-The default implementation allows you to define permissions using config in `application.properties`. An example
-config is shown below:
+.{project-name} policies summary
+|===
+s| Built-in policy s| Description
+s| `deny` | This policy denies all users.
+s| `permit` | This policy permits all users.
+s| `authenticated` | This policy permits only authenticated users.
+|===
 
+It is possible to define role-based policies that will only allow users with the specified roles to access the resources.
```

Comment:
Maybe
```suggestion
You can define role-based policies that allow users with specified roles to access the resources.
```

Reply:
- "It is possible to" > "You can" follows IBM Style guidance to start using "you" more. (Having a warmer more personal tone.)
- Avoid future tense "will" in tech writing. (Tricky to localize in some languages.)
- Place "only" as close as possible to the object it modifies (e.g. "allow only users"), or drop it if possible.